### DISCUSSION TITLE
HifiBerry DAC2 HD is not recognized

### DISCUSSION THREAD

**Discussion by MaschineBrennt** [rern/rAudio#1124] [Fixed Issues]
Hello to all of you.

Setup: Raspberry Pi 4b (4GB ram) + HifiBerry DAC2 HD.

I'm trying to make rAudio (latest release, 20221123) work but It can't recognize the "HifiBerry DAC2 HD" in any way:

if I select "HifiBerry DAC2 HD" in "Audio-I2S" (Settings->System->GPIO Devices section), then consequently (after reboot also) disappear various entries from the "Player" (Settings->Player) page, in particular disappear "Output", "Bit-Perfect" and "Volume"; only "Options" and "Music Player Daemon" remain.

The strange thing is that moOde audio player (8.2.2) recognizes and uses this "HifiBerry DAC2 HD" perfectly (with "DT overlay"="hifiberry-dacplushd").

Please any suggestions/help?

Thanks for your attention

**Comment by MaschineBrennt** [comment]
I've searched around a bit but didn't understand much about it:
[5.15 kernel break Hifiberry DAC 2 HD Hat](https://github.com/raspberrypi/linux/issues/4898)

**Comment by Vseason** [comment]
Hmm. My own sound is via a UGREEN External USB Sound Card Adapter (which shows up as “KT USB Audio” in Output > Device), but if it’s a core output difficulty  this might also conceivably explain the noise issues I’ve seen in 1123.

**Comment by MaschineBrennt** [comment]
Any future possibility that the "rAudio/HifiBerry DAC2 HD" thing can be solved?


Thanks

**Reply by ool99** [reply]
I had the same problem. Solution: Set Audio - I2S to HifiBerry DAC2 HD. Open EEPROM setting by clicking on (?) next to GPIO Devices. Click on the wheel and disable auto EEPROM read. After reboot DAC2 HD shows up as an output device.

**Reply by MaschineBrennt** [reply]
It worked!!!  :-)

Thank you very much, ool99
